### PR TITLE
Query ranking fixes

### PR DESCRIPTION
**Pull request by emaohn**
**NOTE: branched off suggestion tests 
- don't suggest places already selected
- Add food suggestion feature
- Add clarifying comments and fix naming

### CODE REVIEW

**Comment by ganewman42** [comment]
If you PR against SuggestionTests, then it will be much easier to review the diff. At the moment I find it a little hard to follow.

**Comment by emaohn** [comment]
Done! Sorry about that.

File: step-capstone/src/components/Pages/Trip.js [hunk 1]
```diff
@@ -6,15 +6,17 @@ import { Grid } from '@material-ui/core'
...
-        timeRange : [new Date(2020,7,15,2,0), new Date(2020,7,15,20,0)],
-        coordinates : {lat:42.3601, lng:-71.0589} 
+    userCategories: ["bakery"],
+    userBudget: 4,
+    radius: "10000",
+    timeRange: [new Date(2020, 7, 15, 2, 0), new Date(2020, 7, 15, 20, 0)],
+    coordinates: { lat: 36.1699, lng: -115.1398 },
+    items: new Set([])
 }
+//"ChIJmx1Uvc3FyIARdp6ftqC7Gd8","ChIJDRyBe_nEyIARH77JCHU27r8","ChIJVcuReVnbyIARXwAzHqxeAnk","ChIJ_YX8a9LGyIARrcojBX4AgtU"
```
Comment (cthamrun) [inline step-capstone/src/components/Pages/Trip.js:19]: Is this comment meant to be left here?
